cityofaustin/atd-moped
# Update event triggers for moped personnel and personnel roles

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/10810

## Testing
**URL to test:** 
https://deploy-preview-964--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

1. Add team members to a team
2. Remove a team member
3. Swap out one team member for a different name
4. Do any other edits you can to a team member

Changing roles are tracked in the activity log, but aren't shown in the UI. You can check that the mutations are working locally by checking http://localhost:9695/console/events/data/activity_log_moped_proj_personnel_roles/processed or querying the db. 

I also have a note about a bug I saw in staging where if you save empty spaces as a project description, you can't edit the description again (ex: https://moped.austinmobility.io/moped/projects/1790). Try saving a description as "      " and you should see the "-" render so you can still edit the field. 


---
#### Ship list
